issues.py

```diff
@@ -36,53 +48,159 @@ def get_issue_dict(alert, project):
     return {"title": title, "body": "\n".join(lines), "labels": ["LGTM"]}
 
 
-@app.route("/", methods=["POST"])
-def issues_webhook():
+def auth_is_valid(signature):
+    if app.debug:
+        return True
 
-    if not app.debug:
+    return hmac.compare_digest(
+        signature, hmac.new(KEY, request.data, "sha1").hexdigest()
+    )
 
-        digest = hmac.new(KEY, request.data, "sha1").hexdigest()
-        signature = request.headers.get("X-LGTM-Signature", "not-provided")
 
-        if not hmac.compare_digest(signature, digest):
-            return jsonify({"message": "Unauthorized"}), 401
+@app.route("/lgtm", methods=["POST"])
+def lgtm_webhook():
+    """Handle POST requests coming from LGTM, and pass a translated request to GitHub"""
+
+    if not auth_is_valid(request.headers.get("X-LGTM-Signature", "not-provided")):
+        return jsonify({"message": "Unauthorized"}), 403
```

Comment:
The JSON values returned in case of errors seem a bit inconsistent. I see both `{"message": "Unauthorized"}` and `{"error": 500}` . LGTM uses the following form `{"code":403, "error" : "Unauthorized"}` . Perhaps you can adopt that too.